remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/transport/triple/TripleClientInvoker.java

```diff
@@ -238,6 +214,76 @@ public void onCompleted() {
         return future;
     }
 
+    private void processSuccess(boolean needDecode, RpcInternalContext context, SofaRequest sofaRequest, Object o, SofaResponseCallback sofaResponseCallback, TripleResponseFuture future) {
+        ClassLoader oldCl = Thread.currentThread().getContextClassLoader();
+        try {
+            RpcInternalContext.setContext(context);
+
+            SofaResponse sofaResponse = new SofaResponse();
+            sofaResponse.setAppResponse(o);
+            if (EventBus.isEnable(ClientAsyncReceiveEvent.class)) {
```

Comment:
Need to confirm whether the log context information content in the ClientAsyncReceiveEvent event is filled.